rclcpp_components/CMakeLists.txt

```diff
@@ -90,6 +90,25 @@ if(BUILD_TESTING)
   set(components "")
   add_library(test_component SHARED test/components/test_component.cpp)
   target_link_libraries(test_component PRIVATE component)
+  # Fake version of this macro so that rclcpp_components_register_node works
+  macro(_rclcpp_components_register_package_hook)
+    set(rclcpp_components_NODE_TEMPLATE ${CMAKE_CURRENT_SOURCE_DIR}/src/node_main.cpp.in)
+    add_library(rclcpp_components::component ALIAS component)
+  endmacro()
+  include(./cmake/rclcpp_components_register_node.cmake)
+  rclcpp_components_register_node(test_component
+    PLUGIN "test_rclcpp_components::DoesNotExist"
```

Comment:
So this test is to load the `test_rclcpp_components`, and make `cannot find classes` situation in `node_main.cpp.in`. and  checking the exception from `std::runtime_error`, right? just checking if my understanding is correct.

Reply:
yes that is correct.